umake/frameworks/games.py

```diff
@@ -25,9 +25,11 @@
 from gettext import gettext as _
 import logging
 import os
+from os.path import join
```

Comment:
This is not needed since you used the full 'os.path.join', so it can be removed

Reply:
If a remove, its cause `import error`

Reply:
:smile: my mistake